docs/api/utils.md

```diff
@@ -139,7 +139,11 @@ objects.
 
 ## `utils.requestAsBrowser` ⇒ `Promise<(http.IncomingMessage|stream.Readable)>`
 
-Sends a HTTP request that looks like a request sent by a web browser, fully emulating browser's HTTP headers.
+**IMPORTANT WARNING:** This function uses an insecure version of HTTP parser by default and also ignores SSL/TLS errors. This is very useful in
+scraping, because it allows bypassing certain anti-scraping walls, but it also makes it vulnerable to attacks. Do not use this function in other than
```

Comment:
BTW looking at https://snyk.io/blog/node-js-release-fixes-a-critical-http-security-vulnerability/, the attack vector is mainly critical for servers, for clients it's not that risky, so perhaps we can tone down this warning a bit.